vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj.filters

```diff
@@ -648,7 +649,6 @@
     <ClInclude Include="ReactHost\UwpReactInstanceProxy.h">
       <Filter>ReactHost</Filter>
     </ClInclude>
-    <ClInclude Include="ReactPropertyBag.h" />
```

Comment:
Why is this removed?

Reply:
It was removed automatically by VS. We do not have such files. The implementation files are IReactPropertyBag.h and IReactPropertyBag.cpp. I never can understand what rules VS uses to change the .filters files.


---
In reply to: [423977037](https://github.com/microsoft/react-native-windows/pull/4877#discussion_r423977037) [](ancestors = 423977037)